examples/seismic/abc_methods/03_pml.ipynb

```diff
@@ -0,0 +1,1446 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "# 3 - PML"
```

Comment:
I'd use the unabbreviated name in the title?

Reply:
Just a mistake! I already corrected this.